pkl-gradle/src/main/java/org/pkl/gradle/spec/JavaCodeGenSpec.java

```diff
@@ -19,6 +19,8 @@
 
 /** Configuration options for Java code generators. Documented in user manual. */
 public interface JavaCodeGenSpec extends CodeGenSpec {
+  Property<Boolean> getGeneratedAnnotation();
+
```

Comment:
Can you move this to `CodeGenSpec` instead? This way, we can add this same flag to the kotlin code generator. And let's rename this to `getAddGeneratedAnnotation`. Same for `JavaCodeGenTask`.